frontend/src/components/RouteTable.jsx

```diff
@@ -414,14 +426,21 @@ function RouteTable(props) {
             rowCount={displayedRouteStats.length}
             columns={columns}
           />
-          <TableBody>
+          <TableBody onMouseLeave={() => onRowHover(null)}>
             {stableSort(displayedRouteStats, order, orderBy).map(row => {
```

Comment:
This is sort of unrelated to your PR. We have this duplicate route id thing going on right now on the back end, and I noticed that it generates a bunch of console warnings due to duplicate TableRow keys.  Also this breaks the spider map's filtering of the table rows.  I noticed that filtering works again if we use unique keys -- so instead of keying on route id, we could just key on row index, like "routerow" + index.
```suggestion
            {stableSort(displayedRouteStats, order, orderBy).map((row, index) => {
```

Reply:
So are we just going to chuck it in here so we wont need a separate PR?

Reply:
Well, the duplicate problem is fixed now, so I guess it’s not needed.  It was mainly just to get the spider filtering working again to make testing this PR easier.